apps/api/src/app/benchmark/benchmark.service.ts

```diff
@@ -64,7 +64,7 @@ export class BenchmarkService {
 
     const benchmarkAssetProfiles = await this.getBenchmarkAssetProfiles();
 
-    const promises: Promise<number>[] = [];
+    const promises: Promise<{ marketPrice: number; date: Date }>[] = [];
```

Comment:
Can you swap to keep the alphabetical order?

Reply:
done